target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -209,6 +209,9 @@ get_status_led() {
 	m4-8M)
 		status_led="m4:blue:status"
 		;;
+	mikrotik,rbm11g)
+		status_led="rbm11g:green:usr"
```

Comment:
Please use `$boardname:green:usr` and merge it with the rbm33g block

Reply:
Yeah, didn't notice that $boardname != $board and thus thought that introducing special characters into the name would not be a good idea. But that thought didn't make a lot of sense. Thanks for clarifying.